config.json

```diff
@@ -824,6 +824,17 @@
       ]
     },
     {
+      "uuid": "7f4d5743-7ab8-42ca-b393-767f7e9a4e97",
+      "slug": "complex-numbers",
+      "core": false,
+      "unlocked_by": "leap",
```

Comment:
Could you please share with us the reasoning behind this decision?
Don't get me wrong, I am not against it, just curios

And it turned out (while testing a new version of Exercism) that non-core exercise should be unlocked by only a core one.
We should revisit the whole track curriculum, perhaps you can help with a fresh view

Reply:
To be completely honest, I was looking at a c# track, it looks somewhat finished, so I figured that their unlocking graph is finished too. If we will be redoing the whole of dependency graph, maybe it warrants separate issue and separate PR?

Reply:
Yeah, this should be addressed in a separate issue\PR.
I didn't have a chance to create it, I'll create it later today
But still, it's better to leave `unlocked_by` empty for (until we'll figure out a list of `core` exercises)